egs/wsj/s5/utils/parallel/run.pl

```diff
@@ -153,6 +158,34 @@
   }
 }
 
+sub pick_or_exit {
+  if($job_pick eq 'all'){
```

Comment:
I think I'm OK with the PR, but can you please add documentation for this function?

Reply:
Thank you for the feedback, please check the documentation I just added.